examples/graphs/node2vec_movielens.py

```diff
@@ -519,7 +519,7 @@ def create_model(vocabulary_size, embedding_dim):
 ]
 
 """
-Get the embeddings of the query movies
+Get the embeddings of the query movies.
```

Comment:
Suggestion (same as in another comment):

- Define in non-code language what `query_movies` is about (the <thing>).

```python
"""
### Find related movies

Define a list with five movies called `query_movies`.
"""

query_movies = [...
...
```

Then,

```python
"""
Get the embeddings of `query_movies`.
"""

(notice the back ticks around `query_movies`, since "query movies" on their may not make sense to some users.)

Reply:
Done